src/components/item_bids/ItemBidForm.js

```diff
@@ -37,12 +52,41 @@ const ItemBidForm = ({ minimum_price, auction_active, submit, success }) => {
                             {errors.price}
                         </span>
                     )}
+                    {
+                        auto_bid && (
+                            <>
+                                <label htmlFor="price">Your Maximum Bid (USD)</label>
+                                <p>You must bid at least ${minimumMaxBid}</p>
+                                <input
```

Comment:
This input element is very similar to the input element for a regular bid. Do you think there's a way to reduce the repetition? ie you could extract a simple functional component and pass in the things that are different between the 2 input elements as props, but that might be overkill. Maybe Jayden or Josh who have more frontend/React experience can speak to that